CHANGELOG.md

```diff
@@ -194,7 +193,10 @@ to update to at least [cumulus-ecs-task:2.1.0](https://hub.docker.com/layers/cum
     support `aws-sdk` v3 changes.
 
 ### Added
-
+- **CUMULUS-3385**
```

Comment:
These entries are in 18.3.1, they should be targeting the unreleased section of the CL

Reply:
https://github.com/nasa/cumulus/pull/3721/commits/9741c0b2e73b90180632c5256c3f08bdf92cfdd1